supplementary/alica_ros1/supplementary_tests/alica_supplementary_test_library/etc/plans/AcmePlan.pml

```diff
@@ -0,0 +1,29 @@
+{
+  "blackboard": [],
+  "comment": "",
+  "entryPoints": [],
+  "frequency": 0,
+  "id": 2338431421577856715,
+  "inheritBlackboard": false,
+  "masterPlan": false,
+  "name": "AcmePlan",
+  "preCondition": null,
+  "relativeDirectory": "",
+  "runtimeCondition": {
+    "comment": "",
+    "conditionString": "",
+    "enabled": true,
+    "id": 4341705583387040,
+    "name": "AcmeRuntimeCondition",
+    "pluginName": "DefaultPlugin",
+    "quantifiers": [],
+    "variables": [],
+    "libraryName": "alica_supplementary_test_library"
```

Comment:
this should not be required. Change the engine to read this from the outer yaml node

Reply:
see comment: https://github.com/rapyuta-robotics/alica/pull/369#discussion_r1049338203